jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/BottomNavigationTests.java

```diff
@@ -47,19 +47,19 @@ public void hideOnScrollTest() {
     @Test
     public void shiftTest() {
```

Comment:
add checking that selected button is actvive (based on class contains "v-btn--active"). The same thing related with wrapper with set of checks all related elements(active and not active)

Reply:
In a selection process Maiia uses iSelector method - in my opinion it is good practice (to use core), to not add wrapper without real necessity, all checks we may add in assertions.